runner/templates/configmap.yaml

```diff
@@ -22,12 +22,42 @@ data:
       name = "{{ template "fullname" . }}"
       url = {{ .Values.gitlabUrl | quote }}
       token = {{ .Values.token | quote }}
-      executor = {{ default "docker" .Values.executor.type | quote}}
+      executor = {{ default "docker" .Values.executor.type | quote }}
```

Comment:
I think it's OK to break compatibility here. 
I will add upgrade notes and make it the major release.